spec/default_spec.rb

```diff
@@ -0,0 +1,69 @@
+require_relative 'spec_helper'
+
+describe 'rsc_tomcat::default' do
+  let(:chef_run) do
+    ChefSpec::ServerRunner.new do |node|
+      node.set['rsc_tomcat']['application_name'] = 'sample'
+      node.set['rsc_tomcat']['listen_port'] = '8080'
+      node.set['rsc_tomcat']['bind_network_interface'] = 'private'
+      node.set['rsc_tomcat']['vhost_path'] = 'www.example.com'
+      node.set['rsc_ros']['file'] = 'sample/sample.war'
+      node.set['rsc_ros']['provider'] = 'aws'
```

Comment:
Is this section supposed to set the node variables when running `kitchen verify`? Because it doesn't seem to be doing that for me. My kitchen run complains that `access_key` is node defined, and that is because I don't have it defined in my ENV from here: https://github.com/RightScale-Services-Cookbooks/rsc_tomcat/pull/9/files#diff-d3fbd21bc6be7ce4531c27a82b2619deR46

So it seems this isn't being pushed to the `kitchen verify` run. I just changed the `.kitchen.yml` to have some junk values and the chef run will then work.

Also using with some junk values will fail though because the s3 bucket its assuming to use isn't 
```bash
Ran /usr/local/bin/ros.rb -t download -c AWS -u abc123 -p xxx -b cdw-demo -f sample/sample.war -d /opt/tomcat/webapps/sample.war -r us-east-1 returned```